hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/compact/schema/CompactSchemaReplicationSlowTest.java

```diff
@@ -85,8 +101,6 @@ public void testSchemaReplication_whenAMemberThrowsRetryableExceptionAllTheTime_
     @Override
     public Config getConfig() {
         Config config = super.getConfig();
-        // Jet prints too many logs during the test
-        config.getJetConfig().setEnabled(false);
```

Comment:
These lines are not related with the problem. It seems implicitly `jet` is re-enabled. But it is ok, it will not affect the test.

Reply:
No, it is actually getting the config from the base class, and I changed the base class to use the `smallInstanceConfigWithoutJetAndMetrics` (See the CompactSchemaReplicationTestBase in this PR), which has Jet disabled. So, there is no change here

Reply:
`jet` is disabled by default, so my comment can be ignored.

Reply:
Ok resolved.